src/Main.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args){
+
+        // HomeTask 1
+        int[] arr2 = {1,4,3,2};
```

Comment:
input array should be dynamic, not static as specified here
try to use Scanner class